packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -626,6 +624,7 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
                 <ComponentWrapper
                     key={index}
                     index={index}
+                    dir={isRTL(this.state.locale) ? Direction.rtl : Direction.ltr}
```

Comment:
Site should have a prop that opts users into this functionality. Otherwise, I think this is essentially still treating the prop as required.

Reply:
Not sure I'm following on this. Might need to sync up in person.